sites/dotnet-guide/contents/tutorials/resharper-essentials/code-formatting/index.md

```diff
@@ -0,0 +1,57 @@
+---
+type: TutorialStep
+date: 2021-03-04
+title: Code Formatting
+technologies: [.net, csharp]
+topics: [editing,ide]
+products: [resharper]
+author: citizenmatt
+subtitle: Format and reformat code for consistency in your solution.
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+    url: https://TODO
+---
+
+### Format and Reformat Code
+
+Code that is formatted consistently is more readable, and more enjoyable to work with. But what is good formatting?
```

Comment:
"But what is good formatting?"

This question broke my brain. Existential and deep.